doc/src/manual/noteworthy-differences.md

```diff
@@ -137,8 +137,7 @@ For users coming to Julia from R, these are some noteworthy differences:
   * Julia does not provide `nrow` and `ncol`. Instead, use `size(M, 1)` for `nrow(M)` and `size(M, 2)`
     for `ncol(M)`.
   * Julia is careful to distinguish scalars, vectors and matrices.  In R, `1` and `c(1)` are the same.
-    In Julia, they can not be used interchangeably. One potentially confusing result of this is that
-    `x' * y` for vectors `x` and `y` is a 1-element vector, not a scalar. To get a scalar, use [`dot(x, y)`](@ref).
+    In Julia, they can not be used interchangeably.
```

Comment:
can not -> cannot

Reply:
Hey @stevengj thanks for taking a look! You may have missed that even though that line is in green, it's not actually part of the change I'm making?